hamilton/data_quality/default_validators.py

```diff
@@ -312,6 +314,19 @@ def name(cls) -> str:
 ]
 
 
+def _append_pandera_to_default_validators():
+    """Utility method to append pandera validators as needed"""
+    try:
+        import pandera_validators
+    except ModuleNotFoundError:
+        logger.info(f'Cannot import pandera from pandera_validators. Run pip install hamilton[pandera] if needed.')
+        return
```

Comment:
This might be clearer?
```suggestion
    try:
        import pandera
    except ModuleNotFoundError:
        logger.info(f'Skipping import of pandera for pandera_validators. Run pip install hamilton[pandera] if needed.')
        return
    else:
        import pandera_validators
```

Reply:
Agreed, that is a little easier to read.